docs/custom-flows/email-password.mdx

```diff
@@ -34,105 +34,106 @@ A successful sign-up consists of the following steps:
 <Tabs items={["Next.js","React","Remix", "Gatsby", "Expo", "JavaScript"]}>
 
 <Tab>
-<CodeBlockTabs options={['Pages Router', 'App Router']}>
-```tsx filename="pages/sign-up/[[...index]].tsx"
-import { useState } from "react";
-import { useSignUp } from "@clerk/nextjs";
-import { useRouter } from "next/router";
-
-export default function SignUpForm() {
-  const { isLoaded, signUp, setActive } = useSignUp();
-  const [emailAddress, setEmailAddress] = useState("");
-  const [password, setPassword] = useState("");
-  const [pendingVerification, setPendingVerification] = useState(false);
-  const [code, setCode] = useState("");
-  const router = useRouter();
-  // start the sign up process.
-  const handleSubmit = async (e) => {
-    e.preventDefault();
-    if (!isLoaded) {
-      return;
-    }
+<CodeBlockTabs options={['App Router', 'Pages Router']}>
```

Comment:
I see all of these are missing `;`. In other parts of the docs, we end all examples with `;`. Keep that convention here for consistency